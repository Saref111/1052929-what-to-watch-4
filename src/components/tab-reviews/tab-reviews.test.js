import React from "react";
import renderer from "react-test-renderer";
import TabReviews from "./tab-reviews.jsx";

const reviews = [{
  name: `Ivan Ivanov`,
  rate: 8.9,
  date: new Date(),
  text: `Lorem ipsum dolor sit amet. Eligendi non quis exercitationem culpa`,
}];

const comments = [{
  user: {
    name: `111`
  },
  comment: `11111`,
  date: `11.12.1992`,
  rating: 8
}];

test(`Snapshot test for TabReviews`, () => {
  const tree = renderer.create(<TabReviews reviews={reviews} comments={comments}/>);

  expect(tree).toMatchSnapshot();
});
