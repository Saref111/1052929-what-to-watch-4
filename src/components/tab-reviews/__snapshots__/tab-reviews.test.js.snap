// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for TabReviews 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Lorem ipsum dolor sit amet. Eligendi non quis exercitationem culpa
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Ivan Ivanov
          </cite>
          <time
            className="review__date"
            dateTime="20-06-14"
          >
            July 14, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8.9
      </div>
    </div>
  </div>
</div>
`;
